src/toolbars/MeterToolBar.cpp

```diff
@@ -234,7 +234,7 @@ void MeterToolBar::Populate()
       mRecordSetupButton = safenew AButton(this);
       mRecordSetupButton->SetLabel({});
       mRecordSetupButton->SetName(_("Record Meter"));
-      mRecordSetupButton->SetToolTip(XO("Input Metering"));
+      mRecordSetupButton->SetToolTip(XO("Record Meter"));
```

Comment:
Could you please also squash first two commits? That will make commit history clearer

Reply:
Thank you for pointing this out. I've squashed the first two commits into one.